django/db/models/base.py

```diff
@@ -572,6 +572,16 @@ def __getstate__(self):
         """Hook to allow choosing the attributes to pickle."""
         state = self.__dict__.copy()
         state['_state'] = copy.copy(state['_state'])
+        # memoryview cannot be pickled, so cast it to bytes and store
+        # separately.
+        _memoryview_attrs = []
+        for attr, value in list(state.items()):
```

Comment:
We should be able to iterate now though `state.items()` directly without making a copy because state is no longer being mutated in the body of the loop.